### PR TITLE
feat: release v1.5.0

### PR DESCRIPTION
**Pull request by pheekus**
### Bug Fixes

* **foxy-nucleon:** ignore rumour updates before service init ([00c0e6f](https://github.com/Foxy/foxy-elements/commit/00c0e6f96eecc12245a998295d272d0f012c266b))
* fix imports and exports ([78d7492](https://github.com/Foxy/foxy-elements/commit/78d7492e5ce1f36919e429f48ee6d07ca830d965))
* **foxy-collection-pages:** make sure empty collections are synced by rumour ([3a059dc](https://github.com/Foxy/foxy-elements/commit/3a059dce577fb3dde9a02d344f9d449454a4dd92))
* **foxy-collection-pages:** re-render on page property assignment ([eb4eb7d](https://github.com/Foxy/foxy-elements/commit/eb4eb7d91f188a500de3ab2bed8156bb903a9103))

### Features

* **foxy-subscription-card:** add support for default slot/template ([568fce9](https://github.com/Foxy/foxy-elements/commit/568fce90df7ab29b99a409be9f490044f50de663))
* **foxy-table:** add support for default slot/template ([90af72b](https://github.com/Foxy/foxy-elements/commit/90af72b6882dc25cb6d55ae76472e2bc09802dc4))

### Performance Improvements

* avoid reacting to rumour updates in emitters ([656c1c4](https://github.com/Foxy/foxy-elements/commit/656c1c45084b60e7148178681dc37d20d30f9c42))

### CODE REVIEW

**Comment by pheekus** [comment]
Waiting on https://github.com/Foxy/foxy-sdk/pull/19 to be merged. Once `@foxy.io/sdk@1.5.0` is on npm, I will update it in this branch and mark this PR as ready for review.

File: src/elements/public/SubscriptionCard/index.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import '@polymer/iron-icons';
 import '@polymer/iron-icon';
+import '../../internal/InternalSandbox/index';
```
Comment (brettflorio) [inline src/elements/public/SubscriptionCard/index.ts:3]: @pheekus This needed here?

File: src/elements/public/Table/index.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import '../../internal/InternalSandbox/index';
```
Comment (brettflorio) [inline src/elements/public/Table/index.ts:1]: @pheekus This one too?

Comment (pheekus) [reply src/elements/public/Table/index.ts:1]: @brettflorio yes these register an internal sandbox element with a shadow dom where templates are rendered.

Comment (brettflorio) [reply src/elements/public/Table/index.ts:1]: Ok great. Just checking :)

**Review by brettflorio — APPROVED** [review]

**Pull request merged**